modin/data_management/functions/groupby_function.py

```diff
@@ -153,10 +153,11 @@ def caller(
             or isinstance(by, pandas.Grouper)
         ):
             by = try_cast_to_pandas(by, squeeze=True)
-            default_func = (
+            default_func = kwargs.get(
+                "default_to_pandas_func",
                 (lambda grp: grp.agg(map_func))
                 if isinstance(map_func, dict)
-                else map_func
+                else map_func,
```

Comment:
Only one question, why was this comma added?

Reply:
black did this. 
now this expression is one of the `.get` function argument, so a comma is required